### DISCUSSION TITLE
Did anyone elses redshift serverless destination just break for no reason?

### DISCUSSION THREAD

**Discussion by meyerovb** [airbytehq/airbyte#38053] [General]
I'm giving up, gonna do everything to s3 and just schedule copies in a proc from now on... worked great but now it's just hanging with no explanation, not even seeing failed queries in sys_query_history, its latest version and test connection works fine.

**Comment by edgao** [comment]
what version of destination-redshift are you using, and what version of redshift are you running? Redshift [patch 181](https://docs.aws.amazon.com/redshift/latest/mgmt/cluster-versions.html#cluster-version-181) broke part of our schema-drift detection code, which should be fixed in destination-redshift 2.5.0

**Comment by meyerovb** [comment]
I assumed latest, but honestly, it doesn’t matter, the write time is so slow that I am switching to S3 anyways because it was eating up too many serverless RPU. I wish you guys would just write all the files to S3 first and then execute a single copy command per table at the end.
